components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/persistence/JDBCSubscriberPersistenceManager.java

```diff
@@ -122,7 +124,7 @@ public void updateSubscriber(PublisherDataHolder holder) throws EntitlementExcep
             PublisherDataHolder oldHolder = getSubscriber(subscriberId, false);
             String updatedModuleName = getUpdatedModuleName(holder, oldHolder);
             PublisherPropertyDTO[] updatedPropertyDTOs = getUpdatedPropertyDTOs(holder, oldHolder);
-            updatedPropertyDTOs = encryptUpdatedSecretProperties(updatedPropertyDTOs);
+            updatedPropertyDTOs = encryptSecretProperties(updatedPropertyDTOs);
```

Comment:
Same variable naming update can be done here. The initial `updatedPropertyDTOs` contains all properties that were simply updated. Second would contain the updated properties with secrets encrypted.